odooconnector_base/models/product_supplierinfo.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
First I had the same thought about supplierinfo. But actually you might need it, even if you are not using any specific scenario (e.g. migration between databases).
Since ``product.supplierinfo`` is in ``addons/product`` I think that there is no problem in keeping it in ``_base`` and considering it at *basic master data*.